Dockerfile

```
@@ -1,14 +1,18 @@
-FROM --platform=$BUILDPLATFORM golang:1.20-bookworm AS build
+FROM golang:1.20-alpine AS build
 
 ARG TARGETOS
 ARG TARGETARCH
+ARG BUILDTARGET=rollout-operator
+
+RUN apk add --no-cache build-base git
 
 COPY . /src/rollout-operator
 WORKDIR /src/rollout-operator
-RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} make
+RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} make ${BUILDTARGET}
 
 FROM alpine:3.18
-RUN apk add --no-cache ca-certificates
+# need gcompat for glibc compatibility of the boringcrypto image
+RUN apk add --no-cache ca-certificates gcompat
```

We don't need gcompat anymore :)
```suggestion
RUN apk add --no-cache ca-certificates
```